src/controls/MenuButtonImageUpload.tsx

```diff
@@ -52,6 +55,7 @@ export interface MenuButtonImageUploadProps
 export default function MenuButtonImageUpload({
   onUploadFiles,
   inputProps,
+  insertImages,
```

Comment:
For consistency with a similar use-case here:
https://github.com/sjdemartini/mui-tiptap/blob/96a1cae53071fbd09a518415dfbcf26f748f59d7/src/controls/ColorPicker.tsx#L131

how about:

```suggestion
  insertImages = insertImagesDefault,
```

(importing with `insertImages as insertImagesDefault` instead of `insertImages as fallbackInsertImages`).

And then no need to update any of the code below (no conditional needed)

Reply:
@lorenries ?